src/state_types/models/context.rs

```diff
@@ -159,7 +159,9 @@ impl<Env: Environment + 'static> Update for Ctx<Env> {
                     None => Effects::none().unchanged(),
                 },
                 // We let the LibraryLoadable model handle this
-                ActionUser::LibSync | ActionUser::LibUpdate(_) => Effects::none().unchanged(),
+                ActionUser::LibSync | ActionUser::LibUpdate(_) | ActionUser::LibItemsByType(_) => {
```

Comment:
this line will stay unmodified once `LibItemsByType` is removed from `ActionUser`